DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -28,6 +28,8 @@ public interface IEventsSheetsService
         Task<IEnumerable<DiscordEvent>> ListEventsAsync();
 
         Task AddResponseForUserAsync(int eventKey, ulong userId, string responseEmoji);
+        Task AddResponseBatchAsync(IEnumerable<ResponseReaction> reactions);
+        Task ClearResponseBatchAsync(IEnumerable<ResponseReaction> reactions);
         Task ClearResponsesForUserAsync(int eventKey, ulong userId);
```

Comment:
Do you still need the non-batch versions? 🙂

Reply:
Nope, removed in 901aa51 🙂

Reply:
I'm going to add this back in, removing it breaks the discord bot. We will be able to replace it with the code @kwunkiuma is working on, but we can't remove it until then 🙂

Reply:
I removed this again in 0ac5f7f because they are still being used. They won't be used after @kwunkiuma has finished his side of the interface, and then we will be able to remove them after that is pulled in 🙂